src/pages/Home.tsx

```diff
@@ -182,7 +182,17 @@ const Home: () => JSX.Element = () => {
             )}
             {(!isMobile || showGraphUi) && (
               <Flex height="100%" width="100%" alignContent="center">
-                <AqiGraph sensorDocId={currentSensorDocId} />
+                {currentSensorDocId ? (
+                  <AqiGraph sensorDocId={currentSensorDocId} />
+                ) : (
+                  <Heading
```

Comment:
Is this heading also removed from the AqiGraph component? Or where was it before?